pipelines/pipeline_katello_lbproxy.yml

```diff
@@ -0,0 +1,238 @@
+          owner: puppet
+          group: puppet
+      - name: copy the prepared puppet certs
+        copy:
+          src: "/tmp/proxycerts/{{ item }}/{{ ansible_fqdn }}.pem"
+          dest: "/etc/puppetlabs/puppet/ssl/{{ item }}/{{ ansible_fqdn }}.pem"
+          owner: puppet
+          group: puppet
+        with_items:
+          - ['certs', 'private_keys', 'public_keys']
+      - name: restore context of /etc/puppetlabs/puppet/ssl/
+        command: restorecon -Rv /etc/puppetlabs/puppet/ssl/
+      when:
+        - inventory_hostname != 'proxy01'
+  roles:
+    - foreman_proxy_content
+    - foreman_installer
+  tasks:
+    - block:
+      - name: create certs for the other proxies
+        command: "puppet cert generate {{ hostvars[item].ansible_fqdn }} --dns_alt_names={{ lbname }}"
+        environment:
+          PATH: "/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin:/root/bin"
+        args:
+          creates: /etc/puppetlabs/puppet/ssl/certs/{{ hostvars[item].ansible_fqdn }}.pem
+        with_items:
+          - proxy02
+      - name: fetch the certs
+        fetch:
+          src: "/etc/puppetlabs/puppet/ssl/{{ item[0] }}/{{ hostvars[item[1]].ansible_fqdn }}.pem"
+          dest: "/tmp/proxycerts/{{ item[0] }}/{{ hostvars[item[1]].ansible_fqdn }}.pem"
+          flat: True
+        with_nested:
+          - ['certs', 'private_keys', 'public_keys']
+          - ['proxy02']
+      - name: fetch the CA cert
+        fetch:
+          src: /etc/puppetlabs/puppet/ssl/certs/ca.pem
+          dest: /tmp/proxycerts/certs/ca.pem
+          flat: True
+      when:
+        - inventory_hostname == 'proxy01'
+    - name: assign organization and location
+      command: "hammer proxy update --name {{ ansible_fqdn }} --organizations 'Default Organization' --locations 'Default Location'"
+      delegate_to: katello01
+
+- hosts:
+  - haproxy01
+  become: yes
+  vars:
+    selinux_state: 'permissive'
+    foreman_proxies:
+      - proxy01
+      - proxy02
+  roles:
+    - selinux
+    - haproxy
+
+- hosts:
+  - client01
+  become: yes
+  vars:
+    lbname: capsule.example.test
+  roles:
+    - etc_hosts
+    - puppet_repositories
+    - katello_repositories
+    - disable_firewall
+  tasks:
+    - name: discover facts from the other machines
+      setup:
+      delegate_to: "{{ item }}"
+      delegate_facts: yes
+      with_items:
+        - haproxy01
+        - katello01
+        - proxy01
+        - proxy02
+    - name: add {{ lbname }} hosts entry
+      lineinfile:
+        dest: /etc/hosts
+        line: "{{ hostvars['haproxy01'].ansible_default_ipv4.address }} {{ lbname }}"
+    - name: add puppet autosign entry
+      lineinfile:
+        path: /etc/puppetlabs/puppet/autosign.conf
+        line: "{{ ansible_fqdn }}"
+      delegate_to: proxy01
+    - name: fetch bootstrap.py
+      get_url:
+        url: https://raw.githubusercontent.com/Katello/katello-client-bootstrap/puppet-server/bootstrap.py
```

Comment:
404, should probably be https://github.com/Katello/katello-client-bootstrap/blob/master/bootstrap.py

Reply:
right, that's what you get when you use an own branch when testing, but publishing with the  patch merged and not checkig :)